src/model-helpers/DocumentList.js

```diff
@@ -182,5 +182,6 @@ export const loadDocumentList = async (session, podUrl) => {
     });
     docList = [];
   }
+
```

Comment:
Let's avoid random whitespace changes like this. They clutter up code reviews.

Reply:
Reverting changes.